pkg/reconciler/route/domains/domains.go

```diff
@@ -29,6 +32,23 @@ import (
 // HTTPScheme is the string representation of http.
 const HTTPScheme string = "http"
 
+// GetAllDomains returns all of the domains (including subdomains) associated with a Route
+func GetAllDomains(ctx context.Context, r *v1alpha1.Route, names []string) ([]string, error) {
```

Comment:
Possible to refactor
https://github.com/knative/serving/blob/4b5e523a2621714f1c88ce991d096d90b90e132c/pkg/reconciler/route/resources/cluster_ingress.go#L88-L96

to leverage this method?

Reply:
Could you please share more thoughts about the refactoring?. For me, these two parts does not look very similar.

Reply:
Follow up?  I'd like to get the fix in.

Reply:
You're right, at first glance it looked similar to `getAllDomains`. They are actually different. 🤦‍♂